### DISCUSSION TITLE
Option to eliminate/select scrolling in Lovelace Panel Mode

### DISCUSSION THREAD

**Discussion by douglampe** [home-assistant/frontend#11304] [Other feature requests]
## The request

Panel Mode seems to be geared towards designing dashboards to be used as a "panel" (such as a touchscreen). However, the default behavior is to size the content to the WIDTH of the screen even if the aspect ratio of the content causes the content to scroll. This is most likely not the desired result for many panel use cases. For example, I have a Picture Elements card that shows the layout of my home and the status of all of my devices. This picture is closer to a 4:3 aspect ratio than the 16:9 aspect ratio of the screens I want to display it on. Therefore, the bottom of the picture is chopped off and I have to scroll to see all of my devices. This could be resolved by allowing the user to select the scroll option (horizontal, vertical, or none). In my case, I would select "horizontal" which would resize to height rather than width. The image would scroll left to right on the mobile app but would easily fit on the screen on 16:9. The "none" option would fit to whichever the smaller size is relative to the aspect ratio (height on 16:9 and width on mobile).

## The alternatives

I could obviously resize the image in my Picture Elements card to match the aspect ratio of the screens I most want to use it on so it always fits to the width. However this will create unnecessary white space on mobile view and I will also have to redo the layout of all of my elements to match the new size. I can also create a custom panel to handle the horizontal scrolling/vertical sizing, but it seems like this is a good feature for panel mode since most users probably want to be able to fit the content on the screen the dashboard is primarily intended for.

## Additional information

N/A

**Comment by jmbinette** [comment]
Bump +1
